ar/cached_info.go

```diff
@@ -0,0 +1,53 @@
+package ar
+
+import (
+	"github.com/go-co-op/gocron"
+	"github.com/sirupsen/logrus"
+	"github.com/warp-contracts/sequencer/config"
+	"time"
+)
+
+var cacheTaskStarted = false
+var blockInfo *BlockInfo
```

Comment:
for now - yes - removing it would break backwards compatibility in the SDK. Of course we can update this in the SDK, but we cannot assure that everyone will update their version.